src/ActivitylogServiceProvider.php

```diff
@@ -4,8 +4,9 @@
 
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\ServiceProvider;
-use Spatie\Activitylog\Models\Activity;
+use Spatie\Activitylog\Contracts\Activity;
 use Spatie\Activitylog\Exceptions\InvalidConfiguration;
+use Spatie\Activitylog\Models\Activity as ActivityModel;
```

Comment:
If you'd prefer this naming convention it's not a problem changing the Interface class name

Reply:
I'm no fan of the `Contract` suffix either, though I feel like it's very important to have clear naming here, as it already proved to be confusing. I personally prefer `Activity` to keep its name, especially with backwards compatibility in mind. If any of you got a better naming suggestion: feel free to share.

Reply:
I also think the model should keep its name.
If we want to follow a rule we can follow psr naming convention (https://www.php-fig.org/bylaws/psr-naming-conventions/) and appending a suffix (ActivityInterface or ActivityContract).

Reply:
But this convention isn't used by Laravel - and because this is a Laravel package I would follow their guideline. And I can't find any Interface in the `illuminate/contracts` package that has a suffix or prefix to avoid name-conflicts with the extending class.
And IDEs do nearly everything these days to avoid problems and show, before opening a file, if it is an Interface, Abstract or Class.

Reply:
@Gummibeer we already had a few discussions about this at Spatie. While I think the topic is worth discussion furthermore, we probably shouldn't do it here. So I'll let the decision to @freekmurze, I think he can better outweigh the value of consistency with Laravel over our naming conventions.

If you'd like to discuss the topic in general, send me a mail: brent@spatie.be, I'd love to talk about it!

Reply:
Even though, like mentioned by @brendt, we suffix things in our client projects, I think that in this case it's better to follow Laravel conventions. So no suffix.